src/en/developer-guide/applications/utilities/create-and-open.md

```diff
@@ -0,0 +1,28 @@
+---
+author: Lukáš Gallovič
+---
+
+# Create And Open - Entity Generator
+
+### This is a tutorial on how to use custom TS code to generate an entity from a different entity.
+You have to add `JS function` to RibbonDiff.xml, so it looks something like this.
+You must use `createAndOpen` function and `talxis_entitygenerator.js` library.
+```
+<JavaScriptFunction FunctionName="createAndOpen" Library="$webresource:talxis_entitygenerator.js">
+<CrmParameter Value="PrimaryControl" />
+<StringParameter Value="ait_GenerateCase" /> (name of my action)
+</JavaScriptFunction>
+```
```

Comment:
Could you please format the xml, so it's more readable? It would also help to add more parts of the ribbondiff.xml. I believe you need button, which needs action, which is calling this javascriptfunction block I believe?

Imagine that you don't want to try/error next time you're doing this, but rather open up this page and set it up in a few minutes. So it would be better to cover all steps necessary.